django_project/changes/models/sponsor.py

```diff
@@ -49,6 +49,28 @@ class Sponsor(models.Model):
         blank=False,
         unique=False)  # there is a unique together rule in meta class below
 
+    sponsorship_contact_name = models.CharField(
+        help_text=_("Full name of sponsorship contact person."),
+        max_length=255,
+        null=True,
+        blank=True)
+
+    sponsorship_contact_title = models.CharField(
+        _('Sponsorship Representative Title'),
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_('Title of the sponsorship representative i.e Treasurer.')
+    )
+
+    sponsorship_office_address = models.TextField(
```

Comment:
What's the difference between sponsorship_office_address and address ( https://github.com/kartoza/projecta/pull/661/files#diff-371604757162da387f77bda08ad26881L64 ) in this model?